pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
Wouldn't that introduce a circular dependency between `vivaldi` and `vivaldi-ffmpeg-codecs`?

In any case, this doesn't seem to work. I don't think it's possible to find the Chromium version without starting the browser and looking at vivaldi://about/ or checking the changelog.